ParquetClassLibrary/Sandbox/MapChunk.cs

```diff
@@ -38,76 +39,76 @@ public class MapChunk
+            return TrySetParquet(in_floorID, in_position, _floorLayer);
         }
 
         /// <summary>
         /// Attempts to update the block parquet at the given position.
         /// </summary>
-        /// <param name="in_block">The new block to set.</param>
+        /// <param name="in_blockID">ID for the new block to set.</param>
         /// <param name="in_position">The position to set.</param>
         /// <returns><c>true</c>, if the block was set, <c>false</c> otherwise.</returns>
-        public bool TrySetBlock(Block in_block, Vector2Int in_position)
+        public bool TrySetBlock(ParquetID in_blockID, Vector2Int in_position)
         {
-            return TrySetParquet(in_block, in_position, _blockLayer);
+            return TrySetParquet(in_blockID, in_position, _blockLayer);
         }
 
         /// <summary>
         /// Attempts to update the furnishing parquet at the given position.
         /// </summary>
-        /// <param name="in_furnishing">The new furnishing to set.</param>
+        /// <param name="in_furnishingID">ID for the new furnishing to set.</param>
         /// <param name="in_position">The position to set.</param>
         /// <returns><c>true</c>, if the furnishing was set, <c>false</c> otherwise.</returns>
-        public bool TrySetFurnishing(Furnishing in_furnishing, Vector2Int in_position)
+        public bool TrySetFurnishing(ParquetID in_furnishingID, Vector2Int in_position)
         {
-            return TrySetParquet(in_furnishing, in_position, _furnishingLayer);
+            return TrySetParquet(in_furnishingID, in_position, _furnishingLayer);
         }
 
         /// <summary>
         /// Attempts to update the collectable parquet at the given position.
         /// </summary>
-        /// <param name="in_collectable">The new collectable to set.</param>
+        /// <param name="in_collectableID">ID for the new collectable to set.</param>
         /// <param name="in_position">The position to set.</param>
         /// <returns><c>true</c>, if the collectable was set, <c>false</c> otherwise.</returns>
-        public bool TrySetCollectable(Collectable in_collectable, Vector2Int in_position)
+        public bool TrySetCollectable(ParquetID in_collectableID, Vector2Int in_position)
         {
-            return TrySetParquet(in_collectable, in_position, _collectableLayer);
+            return TrySetParquet(in_collectableID, in_position, _collectableLayer);
         }
 
         /// <summary>
         /// Attempts to update the parquet at the given position in the given layer.
         /// </summary>
         /// <param name="in_position">The position to clear.</param>
         /// <returns><c>true</c>, if the parquet was removed, <c>false</c> otherwise.</returns>
-        private bool TrySetParquet(ParquetParent in_parquet, Vector2Int in_position, object[,] in_parquetLayer)
+        private bool TrySetParquet(ParquetID in_parquetID, Vector2Int in_position, ParquetID[,] in_parquetLayer)
```

Comment:
Praise: Seeing the change on this line, from object[,] to ParquetId[,], tells me that this is a good change.

Reply:
Yay, thanks!!